G/google_sparsehash/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "google_sparsehash"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/sparsehash/sparsehash-c11.git", "fabeb799894db0762b6bcc0f1fd66937cb9bc037")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p sparsehash-c11/build && cd sparsehash-c11/build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . -j${nproc}
+cmake --install .
+install_license ../LICENSE 
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
If this is header-only, you likely want to use
```suggestion
platforms = [AnyPlatform()]
```
See https://docs.binarybuilder.org/stable/building/#Platform-independent-packages

Reply:
@Andersen98 :point_up: